src/Shout/SHRBTextStyler.class.st

```diff
@@ -14,7 +14,8 @@ Class {
 		'font',
 		'isForWorkspace',
 		'pixelHeight',
-		'workspace'
+		'workspace',
+		'plugins'
```

Comment:
Same as above for all changes in this file.
I don't think this should be in the PR, right?

Unless the PR is about pluggable highlighting, not about parse plugin mechanism.